.gitignore

```diff
@@ -3,6 +3,7 @@ xcuserdata
 *.xccheckout
 ResourceApp/rswift
 R.generated.swift
+R.watchOS.generated.swift
```

Comment:
Good point. When I built this, my fork was not up to date with `master`, so tvOS was not yet included, so I took my own approach. I'll change it.